src/components/pages/maintenance/maintenance.js

```diff
@@ -150,7 +150,7 @@ export class Maintenance extends Component {
   onTimeIntervalChange = (timeInterval) => this.setState({ timeInterval }, () => this.getData());
 
   render() {
-    const { rulesEntities, deviceEntities, rulesIsPending, t, history } = this.props;
+    const { rulesEntities, deviceEntities, rulesIsPending, theme, t, history } = this.props;
```

Comment:
At what point should these be on separate lines? I recall being asked to make separate lines when lists get long. I've been using 3 as a rule of thumb.

Reply:
This rule isn't strictly enforced but in the past we have been using character count as the cut off. I think the cutoff is 90 something chars. It is a judgment call though and I'm fine with this (currently 86 chars long anyway).